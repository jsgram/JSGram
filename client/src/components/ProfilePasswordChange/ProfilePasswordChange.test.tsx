import React from 'react';
import ProfilePasswordChangeContainer, { ProfilePasswordChange } from './index';
import configureStore from 'redux-mock-store';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import { mount, shallow } from 'enzyme';

describe('Profile password change component', () => {
    let renderer: any;
    let store: any;

    const props = {
        username: 'some value',
        changeProfilePassword: jest.fn(() => ('some value')),
        handleSubmit: jest.fn(() => ('some value')),
        submitting: jest.fn(() => ('some value')),
    };

    beforeEach(() => {
        store = configureStore()(props);
        renderer = shallow(<ProfilePasswordChange {...props}/>);
    });

    test('render - success', () => {
        expect(renderer).toMatchSnapshot();
    });
});
