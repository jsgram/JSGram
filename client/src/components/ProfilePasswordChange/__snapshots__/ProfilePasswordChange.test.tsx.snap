// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile password change component render - success 1`] = `ShallowWrapper {}`;
